gmso/tests/test_top.py

```diff
@@ -15,10 +15,10 @@ def test_write_top(self, typed_ar_system):
         write_top(top, 'ar.top')
 
 
-    def test_pmd_loop(self, typed_ar_system):
-        top = typed_ar_system
-        write_top(top, 'ar.top')
-        pmd.load_file('ar.top')
+    def test_pmd_loop(self, typed_ar_system, typed_water_system):
```

Comment:
Would it be more appropriate to use here: 

```
pytest.mark.parametrize('top', [typed_ar_system, typed_water_system])
def test_pmd_loop(self, top):
```

Reply:
The way it would actually be done is:
```
@pytest.mark.parametrize('top', ['typed_ar_system',
typed_water_system'])
def test_pmd_loop(self, top, request):
    write_top(request.getfixturevalue(top), 'system.top')
    pmd.load_file('system.top')
```

I don't think this is actually cleaner, but it ensures that the water system and ar system get tested separately.

Reply:
Thank you for figuring out the syntax!

I disagree that it's not cleaner; I expect that this structure makes it easier to add other fixtures to this loop since we don't need to re-write the internals of the function